patches/src/main/resources/youtube/settings/host/values/arrays.xml

```diff
@@ -148,8 +148,10 @@
         <item>@string/revanced_overlay_button_play_all_type_entry_10</item>
         <item>@string/revanced_overlay_button_play_all_type_entry_11</item>
         <item>@string/revanced_overlay_button_play_all_type_entry_12</item>
+        <item>@string/revanced_overlay_button_play_all_type_entry_13</item>
     </string-array>
     <string-array name="revanced_overlay_button_play_all_type_entry_values">
+        <item>ALL_CONTENTS_WITH_TIME_ASCENDING</item>
```

Comment:
In `revanced_overlay_button_play_all_type_entries`, new item is added at the last index.
In `revanced_overlay_button_play_all_type_entry_values`, new item is added at the first index.

In this case, key and value will be entered incorrectly.